packages/cli/src/s3-store.ts

```diff
@@ -85,11 +85,11 @@ export class S3Store implements IKVStore {
 
   async find(params?: StoreSearchParams): Promise<Array<IKVStoreFindResult>> {
     const store = await this.#storeMap.get(params?.useCaseName)
-    const dataArray = await toArray(
-      store.createReadStream({
-        limit: params?.limit,
-      })
-    )
+    const options = {
+      limit: params?.limit,
+    }
+    if (params.gt) (options as any).gt = params.gt
+    const dataArray = await toArray(store.createReadStream(options))
```

Comment:
Ah, no, wait! We don't want to use `Partial` here and pass it to `store.createReadStream` because:
1. We don't want to pass the `useCaseName` to `store.createReadStream`
2. `StoreSearchParams` is a type defined specifically for the params passed to the search methods in ikv stores, so I think it's better to rely on the (non-)optionality of particular fields as defined in the type itself rather than make them all optional with `Partial`

Does that make sense, @ukstv ?